lms/static/sass/views/_shoppingcart.scss

```diff
@@ -1,336 +1,194 @@
-        margin-left: ($baseline*0.75);
-        font-weight: bold;
-      }
-    }
-
-    .cart-headings {
-      height: 35px;
-      border-bottom: 1px solid #BEBEBE;
-
-      th {
-        text-align: left;
-        border-bottom: 1px solid $border-color-1;
-
-        &.qty {
-          width: 100px;
-        }
-
-        &.u-pr {
-          width: 70px;
-        }
-
-        &.prc {
-          width: 150px;
-        }
-
-        &.cur {
-          width: 100px;
-          text-align: center;
-        }
-
-        &.dsc{
-          width: 640px;
-          padding-right: 50px;
-          text-align: left;
-        }
-      }
-    }
-
-    .cart-items {
-      td {
-        padding: ($baseline/2) 0;
-        position: relative;
-        line-height: normal;
-
-        span.old-price {
-          position: relative;
-          text-decoration: line-through;
-          color: red;
-          font-size: 12px;
-          top: -1px;
-          margin-left: 3px;
-        }
-      }
-
-      td:nth-child(3) {
-        text-align: center;
-      }
-
-      td:last-child {
-        width: 50px;
-        text-align: center;
-      }
-
-      td:nth-child(1) {
-        line-height: 22px;
-        text-align: left;
-        padding-right: $baseline;
-      }
-    }
-
-    .cart-totals {
-      td {
-        &.cart-total-cost {
-          font-size: 1.25em;
-          font-weight: bold;
-          padding: ($baseline/2) 25px;
-        }
-      }
-    }
-  }
-
-  table.order-receipt {
-    width: 100%;
-
-    .order-number {
-      font-weight: bold;
-    }
-    .order-date {
-      text-align: right;
-    }
-    .items-ordered {
-      padding-top: ($baseline*2.5);
-    }
-
-    tr {
-    }
-
-    th {
-      text-align: left;
-      padding: 25px 0 15px 0;
-
-      &.qty {
-        width: 50px;
-      }
-
-      &.u-pr {
-        width: 100px;
-      }
-
-      &.pri {
-        width: 125px;
-      }
-
-      &.curr {
-        width: 75px;
-      }
-    }
-
-    tr.order-item {
-      td {
-        padding-bottom: ($baseline/2);
-
-        span.old-price {
-          text-decoration: line-through !important;
-        }
-      }
-    }
-  }
+  margin-bottom: $baseline;
+  border-bottom: 1px solid $pink;
+  padding: ($baseline/2) 0;
+  background: $card-errors-background-color;
+  color: $pink;
+  font-family: $sans-serif;
 }
 
 #expiry-msg {
-  padding: 15px;
+  @extend %t-icon6;
+  margin-top: ($baseline*0.15);
+  border-top: 1px solid $expiry-border-color;
   background-color: $gray-l5;
-  margin-top: 3px;
-  font-family: $sans-serif;
-  font-size: 14px;
+  padding: ($baseline*0.75);
   text-shadow: 0px 1px 1px $white;
-  border-top: 1px solid #f0f0f0;
+  font-family: $sans-serif;
 }
-.confirm-enrollment {
 
+.confirm-enrollment {
   .title {
-    font-size:24px;
-    border-bottom:1px solid $gray-l5;
-    text-align: left;
-    line-height:70px;
+    @include text-align(left);
+    @include line-height(47.3);
+    @extend %t-icon3;
```

Comment:
I would use one of the other _typography.scss extends here under the "titles" section instead of using the "icons" sizing extends. (thanks for referencing using _typography.scss though!)